app/form_models/offender_sar_case_form.rb

```diff
@@ -106,7 +107,14 @@ def subject_type
 
   def check_custom_validations_for_step(step)
     @case.validate_date_of_birth if step == "subject-details"
-    @case.validate_third_party_names if step == "requester-details"
+    if step == "requester-details"
+      @case.validate_third_party_names
+      @case.validate_third_party_relationship
+    end 
+    if step == "recipient-details"
+      @case.validate_recipient
+      @case.validate_third_party_relationship
+    end 
```

Comment:
I think we should consider making this a case statement like I think we've done in the `params_for_step` method - not a blocker for now but nice to have :)